### ISSUE TITLE
[OSX] Not installable via easy_install

### ISSUE THREAD

**Issue by jaraco** [jaraco/keyring#1]
```
$ easy_install keyring
Searching for keyring
Reading http://pypi.python.org/simple/keyring/
Reading http://keyring-python.org/
Download error: (8, 'nodename nor servname provided, or not known') -- Some packages may not be found!
Best match: keyring 0.1.macosx-10.3-fat
Downloading http://pypi.python.org/packages/2.6/k/keyring/keyring-0.1.macosx-10.3-fat.tar.gz#md5=8d9240e1ecaf29ec2618312a761d9698
Processing keyring-0.1.macosx-10.3-fat.tar.gz
error: Couldn't find a setup script in /var/folders/0r/0rvdcAjFF+iAIHS44pixXE+++TI/-Tmp-/easy_install-QwZhy2/keyring-0.1.macosx-10.3-fat.tar.gz
$
```

---
- Bitbucket: https://bitbucket.org/kang/python-keyring-lib/issue/1
- Originally reported by: Christophe Simonis
- Originally created at: 2009-08-21T20:20:53.839

**Comment by jaraco** [comment]
Bug confirmed, I've removed binary distribution for Mac from PyPI.
It seems that the setuptools did not handler the bdist_dump distribution correctly. I’ll check this.

---

Original comment by: Kang Zhang

**Comment by jaraco** [comment]
Bug is closed since v0.2 is released, and binary package for OSX is canceled in this version. Binary package for OSX will be published when the upstream Distribute project start supporting for OSX binary executable format.

---

Original comment by: Kang Zhang
